### ISSUE TITLE
Annoying popups

### ISSUE THREAD

**Issue by sciurius** [keybase/keybase-issues#3197]
My linux desktop is always on and logged in.
Since a couple of days, [I presume since a keybase update was applied] each morning I find a couple of pop-ups "You accessed a public folder" on my screen. This may be the result of some file system scan that is executed nightly, but I never got these pop-ups, and --scaringly-- some of the pop-ups involve people that I totally do not know.

What is going on?

**Comment by strib** [comment]
@sciurius: you likely have something that scans your home directory (like a backup program) which isn't configured to skip over mountpoints? So it ends up scanning the `public` directory, causing popups.  Does that sound possible?  Can you configure that program to exclude mountpoints?  For example, duplicity has a `--exclude-other-filesystems` flag.

**Comment by sciurius** [comment]
Yes, I do have several file scanning tools that run nightly. But I never got the pop-ups until very recently. Did keybase change this?

**Comment by strib** [comment]
Yes, we moved the true mountpoint on Linux from `/keybase` to `~/.local/share/keybase/fs`, in order to fix a security issue.  We are considering moving it elsewhere, as problems like this keep showing up.
